### PR TITLE
List scheduled, published texts on booked days

### PR DESCRIPTION
**Pull request by patrickvenetz**
Bildschirmfoto 2023-01-08 um 18 33 14

### CODE REVIEW

File: components/Layout/Container.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const router = useRouter()
+
+  useEffect(() => {
+    setTab(tabs.findIndex((tab) => tab.route === router.pathname))
+  }, [router.pathname])
+
+  const createRoute = (href) => (e) => {
+    e?.preventDefault()
+    router.push(href)
+  }
```
Comment (trm217) [inline components/Layout/Container.js:31]:
```suggestion
```
See below

File: components/Layout/Container.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <meta name='robots' content='noindex,nofollow' />
+      </Head>
+      <div {...styles.tabsContainer}>
+        <Scroller activeChildIndex={tab}>
+          {tabs?.map(({ text, route }, index) => (
+            <TabButton
+              key={`nav-tab-${index}`}
+              text={text}
+              isActive={tab === index}
+              onClick={createRoute(route)}
```
Comment (trm217) [inline components/Layout/Container.js:50]:
```suggestion
              onClick={() => router.push(route)}
```
No need for that extra function

File: .env [hunk 3]
```diff
@@ -1,3 +1,6 @@
-NEXT_PUBLIC_SIGNIN_URL=http://localhost:3010/anmelden
 NEXT_PUBLIC_API_URL=http://localhost:5010/graphql
 NEXT_PUBLIC_CALENDAR_SLUG=vorlesen
+NEXT_PUBLIC_SIGNIN_URL=http://localhost:3010/anmelden
+
+NEXT_PUBLIC_FRONTEND_ENDPOINT=http://localhost:3010
+NEXT_PUBLIC_PUBLIKATOR_ENDPOINT=http://localhost:3005
```
Comment (trm217) [inline .env:6]: Why no .env.example?

File: components/Me/index.js [hunk 4]
```diff
@@ -1,28 +1,13 @@
...
-  const { data, loading: isLoading } = useQuery(GET_ME, options)
-
-  const { me } = data || {}
+  const { me, loading } = useMe()
 
   return (
     <Interaction.P>
-      {isLoading && <>Verbinde ...</>}
-      {!isLoading && !me && (
+      {!me && (
```
Comment (trm217) [inline components/Me/index.js:10]:
```suggestion
      {!loading && !me && (
```

**Review by trm217 — COMMENTED** [review]
Was rather preoccupied this afternoon after all, therefore I wasn't able to thoroughly look at the code. On first glance however, I couldn't find much more than tiny mimis.
Looks good to me (with typescript it would look better though 😉)

File: .env (hunk 3 above)
Comment (patrickvenetz) [reply .env:6]: Renamed .env to .env.example

**Comment by patrickvenetz** [comment]
@trm217 Thanks for your kind review

**Pull request merged**